gopher/stats.py

```diff
@@ -44,9 +66,28 @@ def rankdata(data):
     return ranked
 
 
-def mannwhitneyu(x, y, alternative="two-sided", use_continuity=True):
-    """Version of Mann-Whitney U-test that runs in parallel on 2d arrays.
-    This is the asymptotic algo only."""
+def mannwhitneyu(
```

Comment:
hummm I think the comment should clarify where numba is getting used (since the function itself is not kit-decorated)

Reply:
Good idea. I'll  change that.